clusters/ztp-siteconfig/skidoo/kustomization.yaml

```diff
@@ -0,0 +1,47 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+generators:
+# Uncomment for SiteConfig CR creation
+# - skidoo-siteconfig.yaml
```

Comment:
What does this do if uncommented?  Would like to understand this.

Reply:
It should be uncommented. The reason it's commented for now is to pass the CI.
We need to install the kustomize plugin in the CI for this to pass